packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -137,16 +129,21 @@ export default class QuanticStandaloneSearchBox extends NavigationMixin(
 
   connectedCallback() {
     registerComponentForInit(this, this.standaloneEngineId);
-    this.addEventListener(
-      'suggestionlistrender',
-      this.handleSuggestionListEvent
-    );
+
+    this.addEventListener('inputvaluechange', this.handleInputValueChange);
+    this.addEventListener('submitsearch', this.handleSubmit);
+    this.addEventListener('showsuggestions', this.handleShowSuggestions);
+    this.addEventListener('selectsuggestion', this.handleSelectSuggestion);
   }
 
   renderedCallback() {
     initializeWithHeadless(this, this.standaloneEngineId, this.initialize);
-    if (!this.isInitialized && !!this.standaloneSearchBox && !!this.input) {
-      this.input.setAttribute('is-initialized', 'true');
+    if (
+      !this.isInitialized &&
+      !!this.standaloneSearchBox &&
+      !!this.quanticSearchBoxInput
+    ) {
+      this.quanticSearchBoxInput.setAttribute('is-initialized', 'true');
```

Comment:
Why are we setting this attribute to true here?

Reply:
this is used for the e2e tests in the standalone searchbox